### DISCUSSION TITLE
How to use RESTFul API modify a record based on an autoid primary key

### DISCUSSION THREAD

**Discussion by jygmysoul** [milvus-io/milvus#33791] [Q&A and General discussion]
How to use RESTFul API modify a record based on an auto-incremented primary key

RequestBody:
{
    "collectionName": "Collection01",
    "data": [
        {
            "RecordID": 450406626139262654,  //this is Primary Key Auto-Increment
            "SudentName": "JJJJJJJJ",
            "SudentScore": "12345678",
            "SudentVector": [
                1,
                2,
                3,
                4,
                5
            ]
        }
    ],
    "dbName": "default",
    "partitionName": "Partition01"
}
ResponseBody:
{
    "code": 1804,
    "message": "fail to deal the insert data, error: invalid parameter[expected=][actual=set primary key but autoID == true]"
}

**Comment by yhmo** [comment]
https://milvus.io/docs/insert-update-delete.md#Upsert-entities
As mentioned in this page  "Upsert operations does not support collections with autoID enabled.", it is not allowed to modify a record with autoid=true.
